pkg/remediator/watch/filteredwatcher.go

```diff
@@ -134,10 +135,10 @@ func (w *filteredWatcher) pruneErrors() {
 
 // addError checks whether an error identified by the errorID has been tracked,
 // and handles it in one of the following ways:
-//   * tracks it if it has not yet been tracked;
-//   * updates the time for this error to time.Now() if `errorLoggingInterval` has passed
+//   - tracks it if it has not yet been tracked;
```

Comment:
Duel of the IDEs

Reply:
I think this is actually a GoFmt version issue. 
I have Go 1.20 installed locally, but the project is compatible back to 1.17.